### DISCUSSION TITLE
Add enqueue.sendParent to enqueueActions

### DISCUSSION THREAD

**Discussion by CodingDive** [statelyai/xstate#4980] [Ideas] [answered]
Thank you for all the work on the newest XState. ❤️  I use it every day and the developer experience is unmatched! 

I often see myself wanting to use `enqueue.sendParent`, but it's missing from the `enqueueActions`

https://github.com/statelyai/xstate/blob/9841fab65f65d76349386b9489413f8fd134ee84/packages/core/src/actions/enqueueActions.ts#L69

My alternative right now would be to do `enqueue.sendTo(self._parent...) or defining an extra event in the machine that I then raise to communicate with the parent.

**Comment by davidkpiano** [comment, marked as answer]
Thanks for the suggestion! Made a quick PR here: https://github.com/statelyai/xstate/pull/4981

**Reply by CodingDive** [reply]
Thank you for the new release! ❤️
